tests/expectations/core/test_expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import pytest
+from contrib.experimental.great_expectations_experimental.expectations.expect_column_values_to_be_present_in_other_table import (
+    ExpectColumnValuesToBePresentInAnotherTable,  # noqa: F401 # needed for expectation registration
+)
+
+import great_expectations as gx
+import great_expectations.exceptions as gx_exceptions
+from great_expectations.compatibility.sqlalchemy_compatibility_wrappers import (
+    add_dataframe_to_db,
+)
+
+
+@pytest.fixture
+def referential_integrity_db(sa):
```

Comment:
These tests look great! I'd like to request one unit test:
- One where we pass in input args and assert that the generated `template_dict` looks like how we expect